Goobi/webapp/uii/includes/metseditor/modals/inc_me_modal_kulturnav.xhtml

```diff
@@ -0,0 +1,169 @@
+<ui:composition
+        xmlns="http://www.w3.org/1999/xhtml"
+        xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+        xmlns:composite="http://xmlns.jcp.org/jsf/composite"
+        xmlns:h="http://xmlns.jcp.org/jsf/html"
+        xmlns:f="http://xmlns.jcp.org/jsf/core"
+        xmlns:x="http://myfaces.apache.org/tomahawk">
+
+    <composite:interface/>
+
+    <composite:implementation>
+        <!-- KULTURNAV SEARCH MODAL -->
+        <div id="kulturnavSearchBox"
+             class="modal"
+             tabindex="-1"
+             role="dialog">
+            <div class="modal-dialog"
+                 role="document">
+                <h:panelGroup
+                        styleClass="modal-content"
+                        id="kulturnavModalContent">
+                    <div class="modal-header">
+                        <h5 class="modal-title">
+                            <h:outputText value="#{msgs.NORM_kulturnavSearch}"/>
+                        </h5>
+                        <button
+                                type="button"
+                                class="close"
+                                data-dismiss="modal"
+                                aria-label="Close">
+                            <h:outputText
+                                    value="&amp;times;"
+                                    escape="false"/>
+                        </button>
+                    </div>
+
+                    <div class="modal-body">
+                        <p>
+                            <h:outputText
+                                    value="#{msgs.NORM_descriptionTextKulturnavSearch}"
+                                    escape="false"/>
+                        </p>
+
+                        <div class="row">
+                            <div class="col-10">
+                                <x:inputText
+                                        id="kulturnavInput"
+                                        styleClass="form-control"
+                                        onfocus="goobiWorkflowJS.fireOnEnter(this.id, 'kulturnavSearchAction')"
+                                        value="#{Metadaten.kulturnavSearchValue}"/>
+                            </div>
+                            <div class="col-2 text-right">
+                                <div class="text-right">
+                                    <h:commandLink
```

Comment:
Please use buttons with jsf:action. This needs to be done to keep WCAG2 accessiblity. Anchors (`<h:commandLink>`) have a different meaning than a button. So we need to be using buttons for everything that triggers an action.

Reply:
I noticed some strange behavior when using a button - when you use "Enter" to search (for the second time) it does not triggers search but rather it automatically selects the first row of the results and close the modal. I will investigate more on this.

Reply:
oh, yes. You will need to set the attribute `type="button"`, otherwise it will be a submit-button and be the default action for the form. Once you have a button like this: `<button type="button">`, everything should work as expected

Reply:
Thanks for the tip. This is now resolved. I found out I was using `<button type="button">` for search button but  `<button type="submit">` for select record button.  Removing those (hence falling back to `<button type="submit">`) solved for issue.  See commit:  https://github.com/intranda/goobi-workflow/pull/23/commits/4fe66f0d98027f95da67b0b37e2659306239fa1b